CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.2.3]
+* Minor fixes.
```

Comment:
Should we include MSAL related fixes from common core? e.g. background task warning fix is relevant to MSAL.